src/TextUI/TestRunner.php

```diff
@@ -1111,50 +1151,53 @@ protected function handleConfiguration(array &$arguments): void
+        $arguments['backupGlobals']                                  = $arguments['backupGlobals'] ?? null;
+        $arguments['backupStaticAttributes']                         = $arguments['backupStaticAttributes'] ?? null;
+        $arguments['beStrictAboutChangesToGlobalState']              = $arguments['beStrictAboutChangesToGlobalState'] ?? null;
+        $arguments['beStrictAboutResourceUsageDuringSmallTests']     = $arguments['beStrictAboutResourceUsageDuringSmallTests'] ?? false;
+        $arguments['cacheResult']                                    = $arguments['cacheResult'] ?? true;
+        $arguments['cacheTokens']                                    = $arguments['cacheTokens'] ?? false;
+        $arguments['colors']                                         = $arguments['colors'] ?? ResultPrinter::COLOR_DEFAULT;
+        $arguments['columns']                                        = $arguments['columns'] ?? 80;
+        $arguments['convertDeprecationsToExceptions']                = $arguments['convertDeprecationsToExceptions'] ?? true;
+        $arguments['convertErrorsToExceptions']                      = $arguments['convertErrorsToExceptions'] ?? true;
+        $arguments['convertNoticesToExceptions']                     = $arguments['convertNoticesToExceptions'] ?? true;
+        $arguments['convertWarningsToExceptions']                    = $arguments['convertWarningsToExceptions'] ?? true;
+        $arguments['crap4jThreshold']                                = $arguments['crap4jThreshold'] ?? 30;
+        $arguments['disallowTestOutput']                             = $arguments['disallowTestOutput'] ?? false;
+        $arguments['disallowTodoAnnotatedTests']                     = $arguments['disallowTodoAnnotatedTests'] ?? false;
+        $arguments['enforceTimeLimit']                               = $arguments['enforceTimeLimit'] ?? false;
+        $arguments['excludeGroups']                                  = $arguments['excludeGroups'] ?? [];
+        $arguments['failOnRisky']                                    = $arguments['failOnRisky'] ?? false;
+        $arguments['failOnWarning']                                  = $arguments['failOnWarning'] ?? false;
+        $arguments['executionOrderDefects']                          = $arguments['executionOrderDefects'] ?? TestSuiteSorter::ORDER_DEFAULT;
+        $arguments['groups']                                         = $arguments['groups'] ?? [];
+        $arguments['processIsolation']                               = $arguments['processIsolation'] ?? false;
+        $arguments['processUncoveredFilesFromWhitelist']             = $arguments['processUncoveredFilesFromWhitelist'] ?? false;
+        $arguments['randomOrderSeed']                                = $arguments['randomOrderSeed'] ?? \time();
+        $arguments['registerMockObjectsFromTestArgumentsRecursively']= $arguments['registerMockObjectsFromTestArgumentsRecursively'] ?? false;
```

Comment:
a space is missing before `=`
after solving that, alignment will be back to original one again

Reply:
I tried something like that in the past, let's see if your suggestion works nicer.